src/PipeSys/Command/DelayedHead.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
This whole class, while useful in testing isn't very well thought out and isn't docblocked fully either. Change?